testdata/hybrid/memcached-operator/config/default/kustomization.yaml

```diff
@@ -9,20 +9,14 @@ namespace: memcached-operator-system
 namePrefix: memcached-operator-
 
 # Labels to add to all resources and selectors.
-#labels:
-#- includeSelectors: true
-#  pairs:
-#    someName: someValue
+#commonLabels:
+#  someName: someValue
 
-resources:
+bases:
```

Comment:
Why these changes in yaml though?

Reply:
this is unnecessary, I will do `make generate` and update these files accordingly.

Reply:
@varshaprasad96 because `resources` is deprectaed in favor of `bases`.
Therefore, the changes occurs because we are moving forward to adopted kustmize v4

Reply:
Modified and it looks fine now.